conjur-env/docker-compose.yml

```
@@ -2,7 +2,21 @@ services:
   conjur-env-builder:
     build: 
       context: .
+    environment:
+      - GOOS=linux
+      - GOARCH=amd64
+      - CGO_ENABLED=0
     volumes:
       - .:/conjur-env
       - ../vendor:/pkg
     command: go build -o /pkg/conjur-env -a -ldflags '-extldflags "-static"' .
+  conjur-win-env-builder:
+    build: ""
```

Fix for Docker Compose V2:
```suggestion
    build:
      context: .
```